### PR TITLE
[PDI-18885] Performance degradation due to opening of XML files.

### PR DESCRIPTION
**Pull request by smmribeiro**
Now, before trying to read the file, the file is checked for existence.

@pentaho-lmartins @ssamora @ppatricio @bcostahitachivantara

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 26m 5s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl core
```
---
:ok_hand: All tests passed!
 
Tests run: **4916**, Failures: **0**, Skipped: **7** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/pdi/job/PR-7672/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 24m 6s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl core
```
---
:ok_hand: All tests passed!
 
Tests run: **4916**, Failures: **0**, Skipped: **7** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/pdi/job/PR-7672/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: core/src/main/resources/org/pentaho/di/core/xml/messages/messages.properties
Comment (pentaho-lmartins) [inline core/src/main/resources/org/pentaho/di/core/xml/messages/messages.properties:5]: Why is there an X before each message?

**Review by pentaho-lmartins — CHANGES_REQUESTED** [review]

File: core/src/main/resources/org/pentaho/di/core/xml/messages/messages.properties
Comment (smmribeiro) [reply core/src/main/resources/org/pentaho/di/core/xml/messages/messages.properties]: Didn't stage the final version.

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 24m 58s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl core
```
---
:ok_hand: All tests passed!
 
Tests run: **4916**, Failures: **0**, Skipped: **7** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/pdi/job/PR-7672/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: core/src/main/resources/org/pentaho/di/core/xml/messages/messages.properties
Comment (smmribeiro) [reply core/src/main/resources/org/pentaho/di/core/xml/messages/messages.properties]: Done (resolving).

**Review by ppatricio — APPROVED** [review]

**Comment by ppatricio** [comment]
i18n implemented as requested & messages fixed as Luis highlighted.
Ok for merging. Waiting for @pentaho-lmartins approval.

**Review by pentaho-lmartins — APPROVED** [review]

**Pull request merged**
